packages/openapi-generator/src/generator.ts

```diff
@@ -73,7 +74,7 @@ export async function generateWithParsedOptions(
   }
 
   if (options.clearOutputDir) {
-    await rmdir(options.outputDir, { recursive: true });
+    await emptyDirSync(options.outputDir);
```

Comment:
```suggestion
    // Use newer fs.rm() if available and fall back to fs.rmdir() if not available.
    // This can be removed if node 14 is the oldest supported version.
    const rm = promisesFs.rm || promisesFs.rmdir;
    await rm(options.outputDir, { recursive: true });
```
[req] Please avoid adding dependencies unless necessary.

Reply:
thank you for your review. I replaced `emptyDirSync` with the `fs` methods.